install.sh

```diff
@@ -22,6 +22,13 @@ else
 fi
 
 C9_DIR=$HOME/.c9
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -d | --dest-dir ) C9_DIR="$2"; shift;;
+  esac
+  shift
+done
```

Comment:
That second shift looks a bit dangerous. And altogether this just seems a little verbose. How about

```
if [[ ${1-} == -d ]]; then
    C9_DIR=$2
    shift 2
fi
```